ts/store/reducers/wallet/wallets.ts

```diff
@@ -266,6 +266,17 @@ export const bPayListVisibleInWalletSelector = createSelector(
     pot.map(bPayListPot, bPayList => bPayList.filter(isVisibleInWallet))
 );
 
+/**
+ * Return a CoBadge list visible in the wallet
+ */
+export const cobadgeListVisibleInWalletSelector = createSelector(
+  [creditCardListVisibleInWalletSelector],
+  (creditCardListPot): pot.Pot<ReadonlyArray<CreditCardPaymentMethod>, Error> =>
+    pot.map(creditCardListPot, creditCardList =>
+      creditCardList.filter(cc => cc.pagoPA === false)
+    )
+);
```

Comment:
As discussed in https://www.pivotaltracker.com/story/show/176642752, we should also add the condition `issuerAbiCode !== undefined`
```suggestion
export const cobadgeListVisibleInWalletSelector = createSelector(
  [creditCardListVisibleInWalletSelector],
  (creditCardListPot): pot.Pot<ReadonlyArray<CreditCardPaymentMethod>, Error> =>
    pot.map(creditCardListPot, creditCardList =>
            creditCardList.filter(
        cc => cc.pagoPA === false && cc.info.issuerAbiCode !== undefined
      )
    )
);
```

Reply:
Solved on [ff295e4](https://github.com/pagopa/io-app/pull/2746/commits/ff295e42f7fee1a25fb674772d3df97846c2aa85)